app/dashboard/sync/tezos.py

```diff
@@ -46,7 +46,8 @@ def get_tezos_txn_status(fulfillment):
 
     tx_response = requests.get(f'{BASE_URL}/operations/{txnid}').json()
 
-    if tx_response:
+    # a valid response will return a list whereas an invalid response will return a dict
+    if tx_response and isinstance(tx_response, list) and len(tx_response) > 0:
```

Comment:
Could we switch this to ensure that `txn_status` is set to expired for this scenarios. ?
That way the txn isn't stuck in pending forever

Reply:
We can, are we sure that after a tx is made it will appear on this API instantly? If there is a delay then we risk setting non-propagated tx's to expired?